resources/run_adf.sh

```diff
@@ -200,6 +235,20 @@ if [[ $? -ne 0 ]]; then
       $MONGO_DOWNLOAD_DIR/bin/mongod --port $MONGOD_PORT --dbpath $MONGO_DB_PATH \
         --logpath $LOGS_PATH/mongodb_test.log --pidfilepath $TMP_DIR/${MONGOD}.pid --fork
     fi
+
+    waitCounter=0
+    while : ; do
+        check_mongod $MONGO_DOWNLOAD_DIR
+        if [[ $? -eq 0 ]]; then
+            break
```

Comment:
I've been meaning to add a message here to log that the process has started.  Would you please add something here and in the existing mongohoused checking code that the check was successful, continuing?